activesupport/lib/active_support/core_ext/enumerable.rb

```diff
@@ -144,8 +144,8 @@ def index_with(default = (no_default = true))
   def many?
     cnt = 0
     if block_given?
-      any? do |element|
-        cnt += 1 if yield element
+      any? do |element, *args|
+        cnt += 1 if yield element, *args
```

Comment:
```suggestion
      any? do |*args|
        cnt += 1 if yield *args
```
Why not?

But this then begs the question of: What about keyword arguments?

Reply:
With the `element` it is more explicit to me, but I do not have a strong opinion on this.
Aren't keyword arguments already working with this change?